cmd/kubeadm/app/cmd/init.go

```diff
@@ -467,6 +485,10 @@ func runInit(i *initData, out io.Writer) error {
 		return errors.Wrap(err, "error getting directories to use")
 	}
 
+	if i.csrOnly {
+		return errors.New("--csr-only can only be used with phase cert")
+	}
+
```

Comment:
the above `if i.csrOnly {` blob is redundant.

Reply:
This is so you can't use `--csr-only` when you're not calling the `phase` individually. I don't want it running as part of a normal `kubeadm init`